src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -124,6 +143,26 @@ private static DesktopElement ElementFromUIAElement(IUIAutomationElement uia)
             return null;
         }
 
+        /// <summary>
+        /// Get DesktopElements from UIAElements.
+        /// </summary>
+        /// <param name="uia"></param>
+        /// <returns>An IEnumerable of <see cref="DesktopElement"/></returns>
+        private static IEnumerable<DesktopElement> ElementsFromUIAElements(IUIAutomationElementArray elementArray)
+        {
+            for (int i = 0; i < elementArray.Length; ++i)
+            {
+                var uiaElement = elementArray.GetElement(i);
```

Comment:
2 thoughts from comparing this code with what's in `ElementFromUIAElement`:
1. Do we need a call to `Marshal.ReleaseComObject` somewhere in this method, perhaps on `elementArray`?
2. If yes, can this loop just call `ElementFromUIAElement` for each item in the array?

Reply:
Updating the code to call ElementFromUIAElement instead of duplicating the logic. The element array is released in the calling function ElementsFromProcessId.